Xamarin.Essentials/Calendars/Calendars.shared.cs

```diff
@@ -47,6 +47,24 @@ public static async Task<CalendarEvent> GetEventAsync(string eventId)
             return await PlatformGetEventAsync(eventId);
         }
 
+        public static Task<CalendarEvent> GetEventInstanceAsync(string eventId, DateTimeOffset instanceDate) => PlatformGetEventInstanceAsync(eventId, instanceDate);
+
+        public static Task<string> CreateCalendarEvent(CalendarEvent newEvent) => PlatformCreateCalendarEvent(newEvent);
```

Comment:
Maybe these names can just be `CreateEvent` as we already know about the calendar part.